src/jira/client/jira-client.ts

```diff
@@ -210,15 +219,42 @@ export const getJiraClient = async (
 					)
 			},
 			repository: {
-				delete: (transformedRepositoryId: TransformedRepositoryId) =>
-					instance.delete("/rest/devinfo/0.10/repository/{transformedRepositoryId}", {
-						params: {
-							_updateSequenceId: Date.now()
-						},
-						urlParams: {
-							transformedRepositoryId
-						}
-					}),
+				delete: async (repositoryId: number, gitHubBaseUrl?: string) => {
```

Comment:
Sorry I know you've tested, but just curious the typing here for repoId is number, while the submit of workflow/deployment are string. Do you know does it matter? Coz that will be used in the `properties` to identify the correct piece of data.